### PR TITLE
Allow mods to place predictions

### LINKED ISSUE #138: now MODs can participate in predictions
not much of a bug, but a feature update..

now mods can 'bet', so we should revert #15

### PR DESCRIPTION
**Pull request by RakSrinaNa**
# Description

Twitch now allows moderators to place bets. When they do they will however not be able to choose an outcome.

image
https://help.twitch.tv/s/article/channel-points-predictions

Fixes #138 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)

# How Has This Been Tested?

- [x] Manually on a channel where I'm a mod. Prediction was placed successfully.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (README.md)
- [x] My changes generate no new warnings
- [x] Any dependent changes have been updated in requirements.txt

### CODE REVIEW

File: TwitchChannelPointsMiner/TwitchChannelPointsMiner.py [hunk 1]
```diff
@@ -207,8 +207,6 @@ def run(self, streamers: list = [], blacklist: list = [], followers=False):
                 self.twitch.load_channel_points_context(streamer)
                 self.twitch.check_streamer_online(streamer)
                 self.twitch.viewer_is_mod(streamer)
```
Comment (RakSrinaNa) [inline TwitchChannelPointsMiner/TwitchChannelPointsMiner.py:209]: Do we keep the info the viewer is a mod or not?

It was used only to not cast predictions but could be useful someday.

Comment (Tkd-Alex) [reply TwitchChannelPointsMiner/TwitchChannelPointsMiner.py:209]: Just comment for the moment :)
We save one request :laughing:

Comment (RakSrinaNa) [reply TwitchChannelPointsMiner/TwitchChannelPointsMiner.py:209]: Feels weird to just comment that part. The field will still be present in the Streamer object but with a potentially wrong value. And if one day it is needed we may spend time to understand why it is wrong.

In my opinion would be better to let it as it is and have useless requests being made, or remove it all together.

Will still comment it for now.

**Comment by RakSrinaNa** [comment]
Tested it and prediction is placed normally like on a channel you're not a mod. Not extra step of "confirmation" required.

**Pull request merged**